auth/default_authorizer.cc

```diff
@@ -305,6 +312,47 @@ future<> default_authorizer::revoke_all(const resource& resource) const {
     });
 }
 
+future<> default_authorizer::revoke_all(const resource& resource) const {
+    if (legacy_mode(_qp)) {
+        co_return co_await revoke_all_legacy(resource);
+    }
+    static const sstring query = format("SELECT {} FROM {}.{} WHERE {} = ? ALLOW FILTERING",
+        ROLE_NAME,
+        _auth_ks_name,
+        PERMISSIONS_CF,
+        RESOURCE_NAME);
+    try {
+        auto group0_guard = co_await _migration_manager.start_group0_operation();
```

Comment:
This is important and deserves a comment. We must start group0 operation before performing the read. This guarantees linearizability of this operation. Yay!

Reply:
comment added